CONTRIBUTING

```diff
@@ -1,26 +1,26 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
+HOW TO CONTRIBUTE TO PATCHES OpenSSL
 ------------------------------------
 
 (Please visit https://www.openssl.org/community/getting-started.html for
 other ideas about how to contribute.)
 
-Development is coordinated on the openssl-dev mailing list (see the
-above link or https://mta.openssl.org for information on subscribing).
-If you are unsure as to whether a feature will be useful for the general
-OpenSSL community you might want to discuss it on the openssl-dev mailing
-list first.  Someone may be already working on the same thing or there
-may be a good reason as to why that feature isn't implemented.
+Development is done on GitHub, https://github.com/openssl/openssl.
 
-To submit a patch, make a pull request on GitHub.  If you think the patch
-could use feedback from the community, please start a thread on openssl-dev
-to discuss it.
+To request new features or report bugs, please open an issue.
 
-Having addressed the following items before the PR will help make the
-acceptance and review process faster:
+To submit a patch, make a pull request on GitHub.  If you are thinking of
+making a large contribution, open an issue for it before starting work, to
+get comments from the community.  Someone may be already working on the same
+thing or there may be reasons why that feature isn't implemented.
 
-    1. Anything other than trivial contributions will require a contributor
+To make it easier to review and accept our pull request, please follow these
+guidelines:
+
+    1. Anything other than a trivial contribution requires a contributor
     licensing agreement, giving us permission to use your code. See
```

Comment:
It's better to use the formulation of [cla.html](https://www.openssl.org/policies/cla.html), which calls it a "Contributor License Agreement (CLA)".